engine/interpreter/src/main/scala/pl/touk/nussknacker/engine/definition/validator/JsonValidatorExtractor.scala

```diff
@@ -1,13 +1,12 @@
 package pl.touk.nussknacker.engine.definition.validator
 
-import pl.touk.nussknacker.engine.api.definition.{JsonValidator, ParameterValidator}
+import pl.touk.nussknacker.engine.api.definition.{JsonParameterEditor, JsonValidator, ParameterValidator}
 
 object JsonValidatorExtractor extends ValidatorExtractor {
 
   override def extract(params: ValidatorExtractorParameters): Option[ParameterValidator] = {
     params.extractedEditor match {
-      // TODO add json editor case
-      // case _ => Some(JsonValidator)
+      case Some(JsonParameterEditor) => Some(JsonValidator)
```

Comment:
IMO would be better to move this case to `FixedValueValidatorExtractor` and rename this class to `EditorBasedValidatorExtractor`.  Then we will have two looking almost identical classes: `AdditionalPropertyEditorValidatorDeterminer` and `EditorBasedValidatorExtractor`, and it will be easier to find this duplication of code. WDYT?

Reply:
done